### PR TITLE
Cargo Supply security entry changes

### PR DESCRIPTION
**Pull request by ManopolisJones**
Changes various entries in the security group in the cargo supply console

- Gives spec ops supplies crate a box of smoke bombs instead of three loose grenades, raises the price from 20 to 50 credits
- Gives the beanbag shells crate a single box of beanbag shells instead of 10 loose shells
- Changes "Weapons crate" name to "Security weapons crate", changes one of the two boxes of flashbangs with a box of teargas grenades
- Added two bullet proof vests to armor crate, raised price from 15 to 20 credits
- Replaces flashbang boxes with teargas grenade boxes in the Riot crate, added one extra box of bolas and removed the three loose handcuffs
- renamed "Ballistic gear" to "Combat shotguns", removed two bullet proof vests (now in armor crate)
- removed box of beanbag shells from the shotgun shells crate (now in beanbag shells crate)

Originally I just wanted to give the riot crate those teargas grenades, as they felt more appropriate to use during a station riot, however it quickly lead to more changes to things in the security category that I felt warranted it's own PR. I feel the two changes most prevalent besides the price hike to the spec ops crate would be the name change and subsequent removal of bullet proof vests from the Ballistics gear crate, and the reallocation of the beanbag shell box from the shells crate to the beanbag shells crate. 

Starting with the ballistics gear crate, I for starters never liked the name in the first place, always felt it too vague. To that end I know it was named that due to it's inclusion of the bullet proof vests you would use to hold your shotguns, which moves us onto the 15 credit armor crate. The armor crate I always felt was too useless, 15 credits for a couple of vests simple named "armor" and a couple of security helmets nobody would ever care to buy for. To that end in order to help mitigate the ambiguity of the ballistics gear crates name and the uselessness of the armor crate contents, I opted to rename the ballistics gear crate to "Combat shotguns crate" and move it's vests to the armor crate. This way, if you want something to hold your new shotguns, you'd choose to go for the armor crate which gives you two varieties of body armor, or something like the Tactical Assault gear if you're a big spender.

As for the beanbag shell crate, I always thought the loose shells in the crate looked sloppy, and you'd just as well print out beanbags for whoever buying them at the lathe then actually order the damn things. With that said, you might ask why remove the beanbag shells from the normal shotgun shell crate? Simply put, Shotgun Darts. I had for a moment considered changing the beanbag shells crate to the "Civilian shells crate" and adding the darts to the crate, but I came to the conclusion that darts are just beanbags that give slight toxic damage instead of slight brute. To that end I thought 'Why even bother having beanbags and darts in the same crate if they're practically the same thing?' That and the fact that stun shells do the beanbag shells job only 10x more effective in the same crate, would usually mean beanbag boxes would just catch dust. I really think we should either change the functionalities of dart shells to be more useful, or just remove them flat out, but in the meantime, if someone wants to use less then lethals without leaving shrapnel from the shells box, this'll incentive them to use dart shells.

**TLDR: Armor crate too useless, ballistics crate name too vague, so why not give the bullet proof vest to the armor crate to make name better? Beanbag and Darts too similar, so why not give beanbag box to it's own crate?**

:cl:
 * tweak: Changes contents of various security crates from the supply console

### CODE REVIEW

**Comment by Eneocho** [comment]
Darts can inject chems right? I've never used them seriously.

**Comment by WindowsErrors** [comment]
no, they never could inject reagents, this isn't TG, shotgun darts used to actually do something but since it's oldcode now that also means it's shitcode

**Comment by Eneocho** [comment]
I've only used them once and they did shit damage and another player said that they inject chems or some shit. Seems he was wrong. Do they do better damage against unarmored players? That's another thing I heard (just use slugs).

**Comment by WindowsErrors** [comment]
toxin damage isn't stopped by any armor that i know of but if you have access to darts i don't know why you wouldn't just use slugs while aiming for the foot

**Comment by ManopolisJones** [comment]
> darts are actually just worse than beanbags because beanbags actually stun while darts do half the damage and just knock down

God, you're not even stunned when you get knocked over by darts? How the hell how could we possibly make darts better without just making them syringe gun rounds for your shotgun? What was their original function before they became defunct oldcode?

**Comment by Eneocho** [comment]
Some ideas to make darts useful or interesting are:
* Strong against armored people but weak against unarmored (it goes through the armor and stays inside you or just pierces through an unarmored guy).
* Strong against unarmored but weak against armored (fucks you up if not armored but gets blocked by even weak armor)
* Make them deliver a dose of a single chem (without the ability to replace the chem), like suxameth (for the stun without sleep).
* Buff the damage against simplemobs so they become hard counters to mob antags and mob-looses.
* Make them leave a fuckton of shrapnel inside so whomever is shot NEEDS surgery to survive the massive bloodloss and brute the shrapnel would cause.
* Special interaction when aiming at the eyes that blinds people like a screwdriver to the eyes would, but stronger.
* Piercing the target.
* Window piercing without breaking or damaging said window.
* The darts themselves do little to no damage but when shot at a tile they leave a caltrop for a decent stun+feet damage if running (ghetto caltrops by wiring/welding 2 rods would be based)
* Highly increased odds of internal bleeding with little brute damage to force the guy to go to medical.
* Does organ damage if targeted at a zone (aimed at the head it causes brain damage, aimed at the chest causes lungs/heart damage, etc) but does little brute otherwise.

**Comment by WindowsErrors** [comment]
Leave it to me to point out the one cloud in the otherwise clear sky

- Armor Piercing Fin Stabilized Discarding 12g Sabogun Dart is probably the most viable path for modernization of this, but the thing with making "armor piercing" ammunition is that people are just gonna end up using it all the time because basically everything in the game has some kind of armor value (especially when squidmoko leaves out that bullshit rune platebody that's got 80% armor protection across-the-board), I remember on beeststion whenever anybody got autorifles, cargo, redshirts, whoever, they'd just instinctively load up with AP for this reason

- I think injecting reagents what they did before or something like that, but then people complained about muh gloral or something like that, you can see that TG went down this route before subsequently locking it behind research for this reason

- I think killing simplemobs is already what normal shotgun rounds are for, shotguns are the station weapon that does the most amount of damage in the shortest amount of time, although it is a somewhat viable modernization path considering how simplemob health is just a really big number that isn't based on any armor values, xenomorphs have 200-300 health, grues have 250 health or something, and the spider queen has 500 health, I've literally magdumped a spider queen with a full extended magazine from a Vector and they'll still be sponging bullets like some kind of Fallout 3 enemy

- Leaving a bunch of shrapnel is already what buckshot does

- The caltrop idea seems somewhat overly defensive in nature and sounds like something that an actual item should be made for beforehand (like the flare shells) unless we're just going for shotgun bolas like I'm thinking of

- Shooting someone for internal bleeding is a rather questionable stance when it comes to less-lethal munitions

All the other points seem more like side-effects that should be tacked on to the dart's main damaging aspect, as always modernize the darts in another pr instead of this one

**Comment by Kurfursten** [comment]
With darts, it seems like this is a solution in search of a problem. You are trying to find a niche for darts in an overcrowded space because we have them in the code and no one can remove things because removal bad. But no form of darts (including raider dart guns) has ever been good, and we don't really have any spare niches for them.

If anything you should just lean in on gimmicky and make them do something weird, like you can load a piece of paper onto each dart and have it stick into walls, glass, machinery, and mobs.

> That's Security Lobby.
> An airlock. It opens and closes.
> A paper stabbed into the door with a dart! **Read it**. **Take it down**.

This is just an example. But in short, WinErr is right on this one - all those other suggestions are already niches of different rounds and we don't need more alternate ways to execute someone. Slugs are great at that.

**Comment by Eneocho** [comment]
I was avoiding the gimmicky ideas on purpose, but if we go for full gimmicky shit, the idea of it working as an actual dart was one of the first that came to mind. A synergy with your (defunct) service themed trader goods, you can fire a shotty with dart shells at the dartboard instead of throwing a normal dart.
